plugins/home/src/homePageComponents/FeaturedDocs/FeaturedDocs.tsx

```diff
@@ -0,0 +1,168 @@
+  responseLimit?: number;
+  /** An optional string to customize sublink text */
+  subLinkText?: string;
+  /** An optional string or ReactNode to customize the card title */
+  title?: React.ReactNode | string;
+};
+
+const useStyles = makeStyles<Theme>(
+  theme => ({
+    docDescription: {
+      fontSize: theme.typography.body1.fontSize,
+      fontWeight: theme.typography.body1.fontWeight,
+      marginBottom: theme.spacing(2),
+      marginTop: theme.spacing(2),
+    },
+    docSubLink: {
+      fontSize: theme.typography.subtitle1.fontSize,
+      fontWeight: theme.typography.subtitle1.fontWeight,
+      lineHeight: theme.typography.subtitle1.lineHeight,
+    },
+    docsTitleLink: {
+      fontSize: theme.typography.h6.fontSize,
+      fontWeight: theme.typography.h6.fontWeight,
+      lineHeight: theme.typography.h6.lineHeight,
+    },
+  }),
+  { name: 'BackstageFeaturedDocs' },
+);
+
+/**
+ * A component to display specific Featured Docs.
+ *
+ * @public
+ */
+export const FeaturedDocs = (props: FeaturedDocsProps) => {
+  const {
+    color,
+    customStyles,
+    emptyState,
+    filter,
+    path,
+    responseLimit,
+    subLinkText,
+    title,
+  } = props;
+  const linkText = subLinkText || 'LEARN MORE';
+  const defaultStyles = useStyles();
+  const styles = customStyles || defaultStyles;
+  const catalogApi: CatalogApi = useApi(catalogApiRef);
+  const {
+    value: entities,
+    loading,
+    error,
+  } = useAsync(async () => {
+    const response = await catalogApi.getEntities({
+      filter: filter,
+      limit: responseLimit || 10,
+    });
+    return response.items;
+  });
+
+  return (
+    <InfoCard variant="gridItem" title={title || 'Featured Docs'}>
+      {loading && <Progress />}
+      {error && <ErrorPanel error={error} />}
+      {entities?.length
+        ? entities.map(d => (
+            <div key={d.metadata.name} data-testid="docs-card-content">
+              <Link
+                className={styles.docsTitleLink}
+                data-testid="docs-card-title"
+                color={color}
+                to={
+                  path ||
+                  `/docs/${d.metadata.namespace || 'default'}/${d.kind}/${
+                    d.metadata.name
+                  }/`
+                }
+              >
+                {d.metadata.title}
+              </Link>
+              <Typography className={styles.docDescription}>
+                {d.metadata.description}
+              </Typography>
+              <Link
+                className={styles.docSubLink}
+                data-testid="docs-card-sub-link"
+                color={color}
+                to={
+                  path ||
+                  `/docs/${d.metadata.namespace || 'default'}/${d.kind}/${
+                    d.metadata.name
+                  }/`
+                }
+              >
+                {linkText}
+              </Link>
+            </div>
+          ))
+        : emptyState || (
```

Comment:
according to this logic `<EmptyState />` will be rendered while the entities are loading. It would be nice if we could show `<EmptyState />` once the data has been loaded and there are no matched entities

Reply:
done!